examples/imdb_fasttext.py

```diff
@@ -21,7 +22,49 @@
 from keras.datasets import imdb
 
 
+def create_ngram_set(input_list, ngram_range=2):
+    """
+    Extract n-grams from a list of integers
+
+    >>> create_ngram_set([1, 4, 9, 4, 1], ngram_range=2)
+    {(4, 9), (4, 1), (1, 4), (9, 4)}
+
+    :param input_list:
+    :param ngram_range: integer
+    :return: set of tuples
+    """
+    return set(zip(*[input_list[i:] for i in range(ngram_range)]))
```

Comment:
I think that the original implementation treats n-gram as elements of `list`. (It may be my misunderstanding)

So I change return type from `set` to `list` in `create_ngram_set`.
As a result, there are very small test accuracy improvements.
In this case, sec/epoch is the same of `set` case in my environment.



```
Using Theano backend.
Using gpu device 0: Tesla K40m (CNMeM is enabled with initial size: 95.0% of memory, cuDNN 5004)
Loading data...
25000 train sequences
25000 test sequences
Pad sequences (samples x time)
X_train shape: (25000, 400)
X_test shape: (25000, 400)
Build model...
Train on 25000 samples, validate on 25000 samples
Epoch 1/5
25000/25000 [==============================] - 13s - loss: 0.6315 - acc: 0.7516 - val_loss: 0.5311 - val_acc: 0.8402
Epoch 2/5
25000/25000 [==============================] - 13s - loss: 0.3966 - acc: 0.9063 - val_loss: 0.3742 - val_acc: 0.8737
Epoch 3/5
25000/25000 [==============================] - 13s - loss: 0.2349 - acc: 0.9509 - val_loss: 0.3075 - val_acc: 0.8898
Epoch 4/5
25000/25000 [==============================] - 13s - loss: 0.1489 - acc: 0.9714 - val_loss: 0.2746 - val_acc: 0.8962
Epoch 5/5
25000/25000 [==============================] - 13s - loss: 0.0976 - acc: 0.9843 - val_loss: 0.2555 - val_acc: 0.9021
```